### PR TITLE
Updated readme

### PR DESCRIPTION
**Pull request by cpq**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -69,30 +58,36 @@ Please take a look at [Smart.js](https://github.com/cesanta/Smart.js)
...
+document are made to actual source files in [src](src) directory.
+
+### Utilities
+
+#### Mutable/growing memory buffer (mbuf)
+
+Source is in [src/mbuf.h](src/mbuf.h) and [src/mbuf.c](src/mbuf.c).
+Implements a memory
+buffer that can append data to the end, or into arbitrary position into the
+buffer, automatically growing the buffer. This structure is used as a
```
Comment (mmikulicic) [inline README.md]: automatically growing the buffer, if needed

File: README.md [hunk 2]
```diff
@@ -69,30 +58,36 @@ Please take a look at [Smart.js](https://github.com/cesanta/Smart.js)
...
+placeholder for the AST and strings.
```
Comment (mmikulicic) [inline README.md]: s/placeholder/backing store/

**Comment by mmikulicic** [comment]
LGTM, module minor comments

File: README.md (hunk 1 above)
Comment (cpq) [inline README.md]: Rephrased.

File: README.md (hunk 2 above)
Comment (cpq) [inline README.md]: Rephrased

**Pull request merged**